components/Pricing.js

```diff
@@ -160,7 +160,16 @@ export default function Pricing({ products }) {
                 <img
                   src="/supabase.svg"
                   alt="supabase.io Logo"
-                  className="h-12 text-primary"
+                  className="h-10 text-primary"
+                />
+              </a>
+            </div>
+            <div className="flex items-center justify-start">
+              <a href="https://github.com" aria-label="github.com Link">
+                <img
```

Comment:
@leerob Don't we already have a GitHub svg in `components/GitHub.js`?

Reply:
Yeah, but it's the Octocat. Probably want the word logo.

Reply:
Ah, nvmnd, this is a different logo, not the one for the "Continue with GitHub" button.